loro-rs/src/json_bk.udl

```diff
@@ -0,0 +1,77 @@
+dictionary JsonSchema{
```

Comment:
Why is this file "_bk"

Reply:
Udl mode of `uniffi` doesn't support enum with value, but `JsonSchema` and `JsonXXXOp` are all enum with value. I made the `export_json_update` return `String` temporarily until someone needs them.